src/fs/metadata_log.cc

```diff
@@ -105,6 +105,7 @@ void metadata_log::write_update(inode_info::file& file, inode_data_vec new_data_
 
 void metadata_log::cut_out_data_range(inode_info::file& file, file_range range) {
     file.cut_out_data_range(range, [&](const inode_data_vec& former, const inode_data_vec& left_remains, const inode_data_vec& right_remains) {
+        // Former is a data_vec that intersected with range, left_remains and right_remains are parts of it that didn't
```

Comment:
This comment is redundant for me.

Reply:
https://github.com/psarna/seastar/pull/96#discussion_r417476741

Reply:
But try to compare these comments -- they tell different things.